wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/bootstrap/AbstractBootstrapper.java

```diff
@@ -28,6 +28,7 @@
  * @author Sergii Leshchenko
  */
 public abstract class AbstractBootstrapper {
+
   private final String machineName;
   private final int bootstrappingTimeoutMinutes;
```

Comment:
Since it's used only by one bootstrapping method (#bootstrap but not #bootstrapAsync), I would move it to parameters of the #bootstrap method. It'd make clear that fact that bootstrap async is not limited by time out of the box.